src/test/java/org/folio/service/orders/OrderReEncumberServiceTest.java

```diff
@@ -5,10 +5,10 @@
 import static org.folio.helper.PurchaseOrderHelperTest.ORDER_PATH;
 import static org.folio.rest.core.exceptions.ErrorCodes.FUNDS_NOT_FOUND;
 import static org.folio.rest.core.exceptions.ErrorCodes.ROLLOVER_NOT_COMPLETED;
-import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgress.RolloverStatus.ERROR;
-import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgress.RolloverStatus.IN_PROGRESS;
-import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgress.RolloverStatus.NOT_STARTED;
-import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgress.RolloverStatus.SUCCESS;
+import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverLog.RolloverStatus.ERROR;
+import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverLog.RolloverStatus.IN_PROGRESS;
+import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverLog.RolloverStatus.NOT_STARTED;
+import static org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverLog.RolloverStatus.SUCCESS;
```

Comment:
Schema should not be updated for MG bugfix, please make a separate PR for schema update, and in this PR - just fixing query

Reply:
Also it is good to write some integration test to verify that query works correctly.

Reply:
I believe he did it :) https://github.com/folio-org/mod-orders/pull/639 with query only, and this one with schema changes